File: js/task3b/S3_FN_02.js
const people = [
  { name: 'Alice', age: 30 },
  { name: 'Bob', age: 25 },
  { name: 'Charlie', age: 35 }
];

console.log("Original array:");
console.log(people);
people.sort((a, b) => a.age - b.age);
console.log("Sorted by age (ascending):");
console.log(people);
console.log("Sorted names:");
people.forEach(person => console.log(person.name));

File: python/task3b/S3_FN_02.py
people = [
  {"name": "Alice", "age": 30},
  {"name": "Bob", "age": 25},
  {"name": "Charlie", "age": 35}
]

print("Original list:")
print(people)

people.sort(key=lambda p: p["age"])
print("Sorted by age (ascending):")
print(people)

print("Sorted names:")
for person in people:
  print(person["name"])
